src/mono/browser/runtime/hybrid-globalization/culture-info.ts

```diff
@@ -111,3 +119,20 @@ function getShortTimePattern (pattern: string): string {
     }
     return pattern;
 }
+
+// wraps all substrings in the format in quotes, except for key words
+// transform e.g. "HH h mm min ss s" into "HH 'h' mm 'min' ss 's'"
+// problem with "h" that can be both hour code and a fixed (quoted) part of the format
+// workaround for "fr-CA" by preventing quoting of "HH"
+function wrapSubstrings (str: string) {
+    const keyWords = [SECONDS_CODE, MINUTES_CODE, DESIGNATOR_CODE, PREFIX_24H];
+    const words = str.split(" ");
```

Comment:
is that always space ?  what about tab or another whitespaces ?

Reply:
we cannot predict what it will be, in the current form this implementation is test driven only and it does not try to predict the future editions to unicode standard. This one can be easily changed to `str.split(/\s+/);`